test/utils/introspection/SupportsInterface.behavior.js

```diff
@@ -87,6 +87,35 @@ const SIGNATURES = {
     'castVoteBySig(uint256,uint8,address,bytes)',
     'castVoteWithReasonAndParamsBySig(uint256,uint8,address,string,bytes,bytes)',
   ],
+  Governor_5_3: [
+    'name()',
+    'version()',
+    'COUNTING_MODE()',
+    'hashProposal(address[],uint256[],bytes[],bytes32)',
+    'getProposalId(address[],uint256[],bytes[],bytes32)',
+    'state(uint256)',
+    'proposalThreshold()',
+    'proposalSnapshot(uint256)',
+    'proposalDeadline(uint256)',
+    'proposalProposer(uint256)',
+    'proposalEta(uint256)',
+    'proposalNeedsQueuing(uint256)',
+    'votingDelay()',
+    'votingPeriod()',
+    'quorum(uint256)',
+    'getVotes(address,uint256)',
+    'getVotesWithParams(address,uint256,bytes)',
+    'hasVoted(uint256,address)',
+    'propose(address[],uint256[],bytes[],string)',
+    'queue(address[],uint256[],bytes[],bytes32)',
+    'execute(address[],uint256[],bytes[],bytes32)',
+    'cancel(address[],uint256[],bytes[],bytes32)',
+    'castVote(uint256,uint8)',
+    'castVoteWithReason(uint256,uint8,string)',
+    'castVoteWithReasonAndParams(uint256,uint8,string,bytes)',
+    'castVoteBySig(uint256,uint8,address,bytes)',
+    'castVoteWithReasonAndParamsBySig(uint256,uint8,address,string,bytes,bytes)',
+  ],
```

Comment:
I'm not really a fan of repeating all this interface, I'd look for a way to reuse it easily

Reply:
I updated it to reuse the interface.